src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/repository/ReferralRepository.kt

```diff
@@ -1,12 +1,49 @@
 package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.repository
 
 import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.data.jpa.repository.Query
 import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.AuthUser
 import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.Referral
 import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.ServiceProvider
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.ServiceProviderSentReferralSummary
 import java.util.UUID
 
 interface ReferralRepository : JpaRepository<Referral, UUID> {
+  @Query(
+    value = """select sentAt,
+			referenceNumber,
+			assignedToUserName,
+			interventionTitle,
+			serviceUserFirstName,
+			serviceUserLastName from (	
+	select
+			cast(r.sent_at as TIMESTAMP WITH TIME ZONE) as sentAt,
+			r.reference_number as referenceNumber,
+      dfc.id as dynamicFrameworkContractId,
+			au.user_name as assignedToUserName,
+			i.title as interventionTitle,
+			rsud.first_name as serviceUserFirstName,
+			rsud.last_name as serviceUserLastName,
+			row_number() over(partition by r.id order by ra.assigned_at desc) as assigned_at_desc_seq		
+	from referral r
+			 inner join intervention i on i.id = r.intervention_id
+			 inner join referral_service_user_data rsud on rsud.referral_id = r.id
+			 inner join dynamic_framework_contract dfc on dfc.id = i.dynamic_framework_contract_id
+			 left join dynamic_framework_contract_sub_contractor dfcsc on dfcsc.dynamic_framework_contract_id = dfc.id
+			 left join referral_assignments ra on ra.referral_id = r.id
```

Comment:
this join should cause each referral to duplicate in case of multiple assignments
having a superseded flag by design would help this a lot

Reply:
I see, that's why the `row_number` was used